process/track/baseBlockTrack.go

```diff
@@ -89,6 +117,10 @@ func (bbt *baseBlockTrack) receivedMetaBlock(headerHandler data.HeaderHandler, m
 	bbt.blockProcessor.processReceivedHeader(metaBlock)
 }
 
+func (bbt *baseBlockTrack) checkBlockBasicValidity(_ data.HeaderHandler) error {
```

Comment:
rename checkBasicBlockValidity. However I do not think this is needed. Interceptors are doing the basic validity.

Reply:
function removed